data_handler/handlers/order_books/myswap/main.py

```diff
@@ -0,0 +1,240 @@
+import asyncio
+import math
+from decimal import Decimal
+from pathlib import Path
+
+from handlers.blockchain_call import func_call
+from handlers.order_books.abstractions import OrderBookBase
+
+import pandas as pd
+
+from handlers.order_books.commons import get_logger
+from handlers.order_books.myswap.api_connection.api_connector import MySwapAPIConnector
+from handlers.order_books.myswap.api_connection.data_collectors import braavos_get_tokens_prices
+
+
+MYSWAP_CL_MM_ADDRESS = "0x01114c7103e12c2b2ecbd3a2472ba9c48ddcbf702b1c242dd570057e26212111"
+
+# The maximum tick value available in liqmap.json.gz from MySwap Data Service
+MAX_MYSWAP_TICK = Decimal("1774532")
+
+
+class MySwapOrderBook(OrderBookBase):
+    """Class for MySwap order book."""
+    DEX = "MySwap"
+    MYSWAP_URL = "https://myswap-cl-charts.s3.amazonaws.com/data/pools/{pool_id}/liqmap.json.gz"
+
+    def __init__(self, token_a: str, token_b: str, apply_filtering: bool = False):
```

Comment:
```suggestion
    def __init__(self, base_token: str, quote_token: str, apply_filtering: bool = False):
```
Can we pls stick to this terminology? 🙏🏼